packages/expo-live-photo/src/LivePhoto.types.ts

```diff
@@ -0,0 +1,121 @@
+import { ViewProps } from 'react-native';
+
+export interface LivePhotoViewProps extends ViewProps {
+  /**
+   * The LivePhoto asset to display.
```

Comment:
In the other props (e.g. callbacks) you referred to them as "the live photo" so let's be consistent
```suggestion
   * The live photo asset to display.
```